examples/java/src/main/java/org/apache/ignite/example/table/MapperExample.java

```diff
@@ -37,23 +40,52 @@ public Integer toColumnType(String cityId) {
     }
 
     public static void main(String[] args) throws Exception {
-        var mapper = Mapper.builder(Person.class)
-                .automap()
-                .map("cityId", "city_id", new CityIdConverter())
-                .build();
 
-        try (IgniteClient client = IgniteClient.builder()
-                .addresses("127.0.0.1:10800")
-                .build()
-        ) {
-            RecordView<Person> view = client.tables()
-                    .table("person")
-                    .recordView(mapper);
+        try (Connection conn = DriverManager.getConnection("jdbc:ignite:thin://127.0.0.1:10800/")) {
 
+            try (Statement stmt = conn.createStatement()) {
 
-            Person myPerson = new Person(2, "2", "John Doe", 40, "Apache");
+                stmt.executeUpdate(
+                        "CREATE TABLE IF NOT EXISTS Person ("
```

Comment:
Let's implement the idea from [this comment](https://github.com/apache/ignite-3/pull/7122#discussion_r2598952107) by removing `IF NOT EXISTS`.